streaming/rrdpush.c

```diff
@@ -870,6 +870,12 @@ static int rrdpush_receive(int fd
 
     if(!strcmp(machine_guid, "localhost"))
         host = localhost;
```

Comment:
This is also a case I don't recall.
If `host == localhost`, then should we accept the metrics?

I think I had implemented this case to support the possibility of remote data collection for parts of a system. But I am not sure this is used anywhere...

Reply:
I don't see any way for `machine_guid` to get the value `localhost`, other than a user explicitly putting `localhost` in `/var/lib/netdata/registry/netdata.public.unique.id`. Agree that it looks like a useless piece of code.

Reply:
I was wondering myself what is that check over there. I assumed is some kind of "hack" to replay some kind of a use case when machine_guid string is manually set to "localhost".

If that is not the case, then it is very likely we can just kill it

Reply:
Yes, remove it please, I approve and you merge.